install.md

```diff
@@ -133,11 +133,22 @@ To start bash in a later session, just search for <q>bash</q>.
 
 </div>
 
-## Step 2: Install Julia 1.3.1 or newer
+## Step 2: Install Julia
 
-Install Julia 1.3.1 or newer [by downloading it from the Julia homepage](https://julialang.org/downloads/),
+OSCAR requires at least Julia 1.3.1, but we recommend running it with the latest stable Julia release,
+which is 1.5.3 at the time this is written.
+
+There are several ways to install Julia:
+
+1. [By downloading it from the Julia homepage](https://julialang.org/downloads/),
 and following their [platform specific instructions](https://julialang.org/downloads/platform/).
 
+2. On macOS, you can also install it via [Homebrew](https://brew.sh): `brew install julia` 
+
+3. The [JILL](https://github.com/johnnychen94/jill.py) project is a python package which allows installing and updating Julia on Linux, macOS, Windows -- this is in particular handy for experienced users who may want to install multiple Julia versions in parallel; but also for beginners it can be convenient as it allows updating the installed Julia version quite easily.
```

Comment:
```suggestion
3. The [JILL](https://github.com/johnnychen94/jill.py) project is a python package which allows installing and updating Julia on Linux, macOS -- this is in particular handy for experienced users who may want to install multiple Julia versions in parallel; but also for beginners it can be convenient as it allows updating the installed Julia version quite easily. (JILL also supports Windows, but as explained in step 1 above, OSCAR does not work under Windows directly, but rather requires the Windows Subsystem for Linux (WSL).)
```